src/components/home/JoinText.tsx

```diff
@@ -0,0 +1,9 @@
+const JoinText = () => {
+  return (
+    <div className="flex w-full justify-center">
```

Comment:
```suggestion
    <div className="flex justify-center">
```
add a custom margin left value, such as ml-[10vw] (please check how it looks this is a random value)